.github/workflows/packaging.yaml

```diff
@@ -15,14 +15,14 @@ on:
       - 'packaging/msi/*'
       - 'packaging/icon@48w.ico'
       - '.github/workflows/msi.yaml'
-      - 'Next.mk'
+      - 'Makefile'
```

Comment:
Since `Next.mk` is removed. And, yes no formatting changes here.

Reply:
good that was drift worth fixing!